patches/gcc/6.3.0/950-bionic-android.patch

```diff
@@ -0,0 +1,17618 @@
+Based on https://github.com/crystax/android-toolchain-gcc-6/commits/master
```

Comment:
It looks like one of the GCC patches affects the search for the libraries - several samples fail in Travis CI testing due to inability to *find* libstdc++. I don't see anything suspicious in other patches, so it is likely this one that breaks GCC.